NorsokSCDLibrary.aml

```diff
@@ -27673,7 +27778,61 @@
         <Attribute Name="Description" AttributeDataType="xs:string" />
         <Attribute Name="TMin" AttributeDataType="xs:duration" />
         <Attribute Name="TMax" AttributeDataType="xs:duration" />
-        <ExternalInterface Name="Y" ID="bd6eae66-43f8-47ee-b0a5-62a39f1897c4" RefBaseClassPath="InterfaceClassLibrary/NorsokSignalClass/Out/DO/Y">
+        <ExternalInterface Name="Y1" ID="bd6eae66-43f8-47ee-b0a5-62a39f1897c4" RefBaseClassPath="InterfaceClassLibrary/NorsokSignalClass/Out/DO/Y">
```

Comment:
I assume this is to solve issue #37 right? Is there any documentation included in the PR to describe which interfaces can be dynamically added and which cannot? It was not clear at first why we would add `Y2` until I figured it's part of the `Y1..Yn` scheme.

Reply:
Issue #36 concludes that supporting documentation will be provisioned in the repository, in version controlled markdown file(s).